teuthology/run.py

```diff
@@ -243,13 +244,26 @@ def report_outcome(config, archive, summary, fake_ctx):
 
     report.try_push_job_info(config, summary)
 
+    teuth_log = os.path.join(archive, 'teuthology.log')
     if passed:
         log.info(status)
+        make_html_log(teuth_log)
     else:
         log.info(str(status).upper())
```

Comment:
I don't see why we couldn't always make the status line uppercase and simplify this block. Thoughts?

Something like:

```
teuth_log = os.path.join(archive, 'teuthology.log')
log.info(str(status).upper())
make_html_log(teuth_log)
if not passed:
    sys.exit(1)
```